pkg/build/node/node.go

```diff
@@ -22,10 +22,11 @@ import (
 	"os"
 	"path"
 	"regexp"
-	"sigs.k8s.io/kind/pkg/util"
 	"strings"
 	"time"
 
+	"sigs.k8s.io/kind/pkg/util"
```

Comment:
aside:
will re-arrange this later, imho this is a more correct place, but still not the most, it should be in the bottom block with our own imports:
- stdlib
- third party
- kubernetes
- kind

import grouping might be something to doc somewhere, I wonder if any of kubernetes's guides cover this...